.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+tasks:
+  - init: npm i && npm run setup
+    command: npm run serve:dev
```

Comment:
I'm adding `NODE_ENV=production node_modules/grunt/bin/grunt webpack` in front of this, so we get all deps built for GitPod!